_styles/main.css

```diff
@@ -410,6 +410,43 @@ footer ul li a:focus {
     outline: none;
 }
 
+/***********
+* Sections *
+***********/
```

Comment:
I do like flexbox.

I tried to keep it as generic as possible so we could reuse it, but I'm not sure if any other page would use it. Maybe the get involved page if we had an image or something to take up a lot of space.

As of right now this could be moved to `home.css`